x-pack/plugins/security_solution/public/management/components/console/components/history_output.tsx

```diff
@@ -19,6 +21,16 @@ export const HistoryOutput = memo<OutputHistoryProps>((commonProps) => {
   const dispatch = useConsoleStateDispatch();
   const getTestId = useTestIdGenerator(useDataTestSubj());
 
+  const historyBody = useMemo(() => {
+    return historyItems.map((historyItem) => {
```

Comment:
Could save a line of code by:
```jsx
return historyItems.map((historyItem) => ( 
       <HistoryItem key={historyItem.id}>
          <CommandExecutionOutput item={historyItem} />
        </HistoryItem>
        )
 );
```

Reply:
Yeah. True. I sometime opt for this block approach in order to make it easier to debug in the Console (can place a breakpoint on the `return `